sources/packages/web/src/views/student/StudentProfile.vue

```diff
@@ -5,6 +5,48 @@
     information needs to be changed please visit
     <a href="http://id.gov.bc.ca" target="_blank">id.gov.bc.ca</a>.
   </Message>
+  <span v-if="showApplyPDButton">
+    <v-btn
+      color="primary"
+      @click="applyPDStatus()"
+      v-if="showApplyPDButton"
+      :disabled="disableBtn"
+    >
+      Apply for PD status
+      <span v-if="disableBtn">
+        &nbsp;&nbsp;
+        <ProgressSpinner style="width:30px;height:25px" strokeWidth="10"
+      /></span>
+    </v-btn>
+  </span>
+  <span v-else>
+    <Message
+      severity="warn"
+      :closable="false"
+      v-if="
+        studentAllInfo?.pdSentDate &&
+          studentAllInfo?.pdUpdatedDate === null &&
+          studentAllInfo?.pdVerified === null
+      "
+    >
+      <strong>PD Status: Pending</strong>
+    </Message>
+    <Message
+      severity="success"
+      :closable="false"
+      v-if="studentAllInfo?.pdVerified === true"
```

Comment:
While demoing it, please remember to mention that students validated through SFAS DB will also be displaying the status in the same way 😉